CMakeLists.txt

```
@@ -233,14 +233,12 @@ elseif( CMAKE_SYSTEM_NAME MATCHES "Darwin" )
 endif()
 
 # Try to get the current commit information
-set( GIT_COMMIT_SHORT "unknown" )
-set( GIT_COMMIT_LONG "unknown" )
-set( GIT_COMMIT_TIME "unknown" )
+set( GIT_DESCRIBE "unknown" )
 find_package( Git QUIET )
 if( GIT_FOUND )
    execute_process(
       COMMAND
-         ${GIT_EXECUTABLE} show -s "--format=%h;%H;%cd" --no-show-signature --no-notes
+         ${GIT_EXECUTABLE} describe --abbrev=5
```

```suggestion
         ${GIT_EXECUTABLE} describe --abbrev=7
```
How about using the same as the default Git short hash length?

---

Sure.